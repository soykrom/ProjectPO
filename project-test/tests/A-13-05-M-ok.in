5
5
1
5
10
0
0
