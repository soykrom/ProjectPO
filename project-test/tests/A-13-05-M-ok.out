Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 1 não se encontra suspenso.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 10 não existe.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 